src/App.js

```diff
@@ -1,16 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
+import HeaderLikeCounter from './components/HeaderLikeCounter';
 
 const App = () => {
+  const [entries, setStatus] = useState(chatMessages);
+  const [likeCount, setLikeCount] = useState(0);
```

Comment:
Because we can determine the number of likes from `entries`, `likeCount` should not be its own state variable. In general, we want state variables to not depend on one another, otherwise we get multiple sources of truth and increase the risk of bugs.